### ISSUE TITLE
Fix `at-rule-descriptor-value-no-unknown` false positives for `@view-transition` `types` descriptor

### ISSUE THREAD

**Issue by ilya-sarantsev** [stylelint/stylelint#9296]
### What minimal example or steps are needed to reproduce the bug?

```
@view-transition {
  types: forward, back;
}
```

As per https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@view-transition `types` can be a list of valid identifiers 

### What minimal configuration is needed to reproduce the bug?

```
export default {
  extends: ['stylelint-config-standard'],
  rules: { "at-rule-descriptor-value-no-unknown": true } 
};
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACA3AljA7gWgC4CcBDAOwGc1c0JiACYAHVptwE8AHGU+GgMwn0yF8UADQ0ARoTABrANyMAviBEgeaADYwAcoQC2cRDAAeetpoB0YUqWXhqagOYIQxtv1w1YPQgFd1HhiZjXBhiKC4aAG0AclJWTXU0YlxsSGJHbDiSKCEoaIBdEUYaGnw-Tm5gGnoQQhSyzWxYUjB8NDZcfmwUQnUfGGxiCGwfYmkhzGIa7gJ+mgVq4gV5Yls0xwAxfl065ziWBKTcS3s0B3NdACsbFVg2G0RAkpr9w+Tp6pB1Os5cGqKmC94jBEslUqcHJlcNlch8at8QnEaopGCAFEA)

### Which Stylelint-related dependencies are you using?

```
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

No problem to be reported

### What actually happened?

`at-rule-descriptor-value-no-unknown` was reported

**Comment by jeddy3** [comment]
@ilya-sarantsev Thanks for the report and for using the template.

Can you confirm that a _comma_ separated list is allowed, please?

The [MDN docs say](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@view-transition#types) "one or more _space_-separated `<custom-ident>`", but that may not reflect the latest specs.

**Comment by Mouvedia** [comment]
> Value: | none \| \<custom-ident\>+

**Comment by jeddy3** [comment]
Closing as stale. Feel free to reopen the issue with a clarification.
